async-nats/src/jetstream/message.rs

```diff
@@ -336,6 +336,10 @@ pub struct Acker {
 // The async-trait crate is not a solution here, as it would mean we're allocating at every ack.
 // Creating separate function to ack just to avoid one duplication is not worth it either.
 impl Acker {
+    /// Public constructor
```

Comment:
Please remove this comment.
It's obviously a public constructor :).

Reply:
Done.